indico/htdocs/js/utils/forms.js

```diff
@@ -78,8 +97,37 @@
         }).on('change input', function() {
             var $this = $(this);
             var untouched = $this.serialize() == $this.data('initialData');
+            var $cornerMessage = $('.corner-message');
             $this.find('[data-disabled-until-change]').prop('disabled', untouched);
             $this.closest('form').data('fieldsChanged', !untouched);
+            if ($this.find('[data-save-reminder-message]').length) {
```

Comment:
I'd just call it `data-save-reminder` since otherwise it sounds like you could specify a custom message.